airbyte-integrations/connectors/source-postgres/src/test/java/io/airbyte/integrations/source/postgres/CdcPostgresSourceLegacyCtidTest.java

```diff
@@ -4,13 +4,33 @@
 
 package io.airbyte.integrations.source.postgres;
 
+import io.airbyte.integrations.source.postgres.PostgresTestDatabase.BaseImage;
+import io.airbyte.integrations.source.postgres.PostgresTestDatabase.ContainerModifier;
 import org.junit.jupiter.api.Order;
 
 @Order(2)
 public class CdcPostgresSourceLegacyCtidTest extends CdcPostgresSourceTest {
 
-  protected static String getServerImageName() {
-    return "debezium/postgres:13-bullseye";
+  @Override
+  protected PostgresTestDatabase createTestDatabase() {
+    return PostgresTestDatabase.in(BaseImage.POSTGRES_13, ContainerModifier.CONF).withReplicationSlot();
+  }
+
+  @Override
+  protected int getPostgresVersion() {
+    return 13;
```

Comment:
Please don't do this. Instead, identify the postgres version using `testdb.getContainer().getDockerImageName()` or something like that.

Reply:
Put the major version into baseImage enum, because some docker image name does not have this info.